### DISCUSSION TITLE
A needed awnser

### DISCUSSION THREAD

**Discussion by JCV67** [Budibase/budibase#13574] [Help]
Hi @aptkingston  I really would like you to awnser my questions on  #13411 that you just locked. Why lock it, never saw this behavior before.

**Comment by shogunpurple** [comment]
@JCV67 They aren’t questions - they are demands. 

Let me be clear - we will **not** be introducing retroactive plans for old self hosted users before the pricing and tier change. It would defeat the purpose of changing it in the first place. Self-hosted users on versions before the limit change are unaffected, and can continue to use budibase with unlimited users.

We’ve explained (at length) the reasons for the change in various places in GH discussions, blog posts and emails.

Locking this discussion, as it’s bordering on spam at this point.
